tox.ini

```diff
@@ -14,6 +15,20 @@ commands =
     python3 ./Tensile/bin/Tensile Tensile/Configs/build_client.yaml {envdir}/client
     py.test -v --basetemp={envtmpdir} --junit-xml={toxinidir}/python_tests.xml --junit-prefix={envname} --color=yes -n 4 --prebuilt-client={envdir}/client/0_Build/client/tensile_client {posargs}
 
+
+[testenv:{py35,py36,py38,py27}-win]
```

Comment:
It looks like this part is added in #1537 (which is not included in 5.3). 
We do not need it for this change.

Reply:
Removed this section